python/lsst/rubintv/production/utils.py

```diff
@@ -72,3 +81,148 @@ def checkRubinTvExternalPackages(exitIfNotFound=True, logger=None):
 
     if exitIfNotFound and (not hasGoogleStorage or not hasEfdClient):
         exit()
+
+
+def getPlotSeqNumsForDayObs(channel, dayObs, bucket=None):
+    """Return the list of seqNums for which the plot exists in the bucket for
+    the specified channel.
+
+    Parameters
+    ----------
+    channel : `str`
+        The channel.
+    dayObs : `int`
+        The dayObs.
+    bucket : `google.cloud.storage.bucket.Bucket`, optional
+        The GCS bucket, created if not supplied.
+
+    Returns
+    -------
+    seqNums : `list` [`int`]
+        Sorted list of ints of the seqNums for which the specified plot exists.
+    """
+    if channel not in CHANNELS:
+        raise ValueError(f"Channel {channel} not in {CHANNELS}.")
```

Comment:
This `raise` isn't discussed in the docstring.

Reply:
Thanks - I've added all of these, changed them all to being `ValueError`s, and made all the docs actually agree with the `raise` too, which I just spotted they didn't before 🙂